cookbook/web_server/built_in.rst

```diff
@@ -41,6 +41,15 @@ can change the socket passing an IP address and a port as a command-line argumen
 
 .. note::
 
+    You can use the ``--force`` option to force the web server start
+    if the process wasn't correctly stopped (without using the server:stop command).
```

Comment:
We should enclose `server:stop` with double backticks, but otherwise this looks really nice.